enclave-manager/web/src/components/AppLayout.tsx

```diff
@@ -0,0 +1,19 @@
+import { Flex } from "@chakra-ui/react";
+import React, { PropsWithChildren } from "react";
+
+type AppLayoutProps = PropsWithChildren<{
+  Nav: React.ReactElement;
+}>;
+
+export const AppLayout = ({ Nav, children }: AppLayoutProps) => {
+  return (
+    <Flex flexDirection={"row"}>
+      {Nav}
+      <Flex as="main" w={"100%"} justifyContent={"flex-start"} p={"20px 40px"} className={"app-container"}>
```

Comment:
Just for my learning: what does the two p values (20, 40) mean?

Reply:
it's the same as setting `padding: 20px 40px` on the styles.